### DISCUSSION TITLE
Failed to open stream: No such file or directory (continuation to #6485)

### DISCUSSION THREAD

**Discussion by TheGreatBeginner777** [firefly-iii/firefly-iii#6496] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [ ] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [ ] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I understood that I should do 2 tasks:

1. position to /var/www/ within CLI and do 
`sudo chown -R www-data:www-data firefly-iii`
`sudo chmod -R 775 firefly-iii`
_Done and and confirmed with `ls-la`_ in folder /var/www/firefly-iii/
2. go to https://domain.tld/firefly/flush
_Done_

What I experience when visiting https://domain.tld/firefly is this message
> This page isn’t working
domain.tld is currently unable to handle this request.
HTTP ERROR 500

### Expected behaviour

Get into Firefly-III.

### Debug information

n/a.

### Additional info

New to this one hence the reason for so much questions and opened issues. My apologies.

**Comment by JC5** [comment]
Not a problem, but let's continue in a discussion threat. How are you running Firefly III?

**Comment by TheGreatBeginner777** [comment]
Yunohst installation. 
Version: 5.7.12~ynh1

**Reply by JC5** [reply]
Mmm, sorry, I know very little about Yunohost myself. I hope somebody else does!
